app/src/main/java/com/fluffybros/tujefe/ui/add/ManualAddFragment.kt

```diff
@@ -10,7 +10,9 @@ import androidx.navigation.fragment.findNavController
 import com.fluffybros.tujefe.MainViewModel
 import com.fluffybros.tujefe.R
 import com.fluffybros.tujefe.databinding.FragmentManualAddBinding
+import kotlinx.coroutines.ObsoleteCoroutinesApi
 
+@ObsoleteCoroutinesApi
```

Comment:
why do we need to add this?

Reply:
references to MainViewModel need to be under some kind of coroutine annotation, the obsoleteCoroutinesApi was the easiest one to implement without causing other effects.